tail.go

```diff
@@ -325,6 +328,8 @@ func (tail *Tail) tailFileSync() {
 			if tail.Err() == errStopAtEOF {
 				continue
 			}
+			// Send a signal to caller asking a shutdown
+			tail.Lines <- &Line{Err: err}
```

Comment:
Skip this na, can just handle channel close

Reply:
need to give this info to caller for logging